src/main/java/org/primefaces/extensions/component/orgchart/DefaultOrgChartNode.java

```diff
@@ -0,0 +1,144 @@
+/**
+ * Copyright 2011-2018 PrimeFaces Extensions
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.primefaces.extensions.component.orgchart;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * <code>orgchart</code> component.
+ *
+ * @author @jxmai / last modified by $Author$
+ * @version $Revision$
+ * @since 6.3
+ */
+public class DefaultOrgChartNode implements OrgChartNode {
+
+    private String id;
+
```

Comment:
Should we implement equals(), hashCode() and toString() just in case someone wants to add these to a hashMap or set to track them or check two nodes equality?

Reply:
I think we could yes :wink: